docs/self-managed/components/orchestration-cluster/identity/initial-setup.md

```diff
@@ -0,0 +1,318 @@
+
+This user will be assigned to the `admin` role and granted all permissions in the system. Once an admin user exists, this screen is no longer shown.
+
+:::note
+This option does not apply to **Camunda 8 Run** and **Docker Compose** installations, as they come with a pre-configured `demo` user by default.
+:::
+
+### Create an initial admin user via the Setup endpoint of Orchestration Cluster REST API {#create-admin-user-via-setup-rest-api}
+
+As a programmatic alternative to the UI, you can create the first admin user by calling the Setup API endpoint. This is useful for automated or headless installations.
+
+- **API endpoint**: `POST /v2/setup/user` (see the [documentation](/apis-tools/orchestration-cluster-api-rest/specifications/create-admin-user.api.mdx))
+
+The following JSON request body can be used:
+
+```json
+{
+  "username": "<your chosen username>",
+  "password": "<your chosen password>",
+  "name": "<the user's full name>",
+  "email": "<the user's email address>"
+}
+```
+
+This endpoint is only available as long as **no user is assigned to the `admin` role**.
+
+### Assign users, clients, groups, or mapping rules to roles via configuration
+
+The Orchestration Cluster provides a number of [built-in roles](/components/concepts/access-control/authorizations.md#default-roles) with predefined permissions for easier setup.
+
+To assign users, clients, groups, or [mapping rules](/components/concepts/access-control/mapping-rules.md) to roles, add the appropriate properties to your `application.yaml` or set them as environment variables.
+
+<Tabs groupId="option" defaultValue="env" queryString values={
+[
+{label: 'Environment variables', value: 'env' },
+{label: 'application.yaml', value: 'application-yaml' },
+{label: 'Helm values', value: 'helm-values' },
+]}>
+
+<TabItem value="env">
+
+```shell
+CAMUNDA_SECURITY_INITIALIZATION_DEFAULTROLES_<role>_USERS_0=<username>
+CAMUNDA_SECURITY_INITIALIZATION_DEFAULTROLES_<role>_CLIENTS_0=<client id>
+CAMUNDA_SECURITY_INITIALIZATION_DEFAULTROLES_<role>_GROUPS_0=<group id>
+CAMUNDA_SECURITY_INITIALIZATION_DEFAULTROLES_<role>_MAPPINGS_0=<mapping id>
+
+# add more members as desired by repeating the variables with an incremented index
+```
+
+</TabItem>
+
+<TabItem value="application-yaml">
+
+```yaml
+camunda.security.initialization.default-roles.<role>.users[0]: <username>
+camunda.security.initialization.default-roles.<role>.clients[0]: <client id>
+camunda.security.initialization.default-roles.<role>.groups[0]: <group id>
+camunda.security.initialization.default-roles.<role>.mappingrules[0]: <mapping id>
+# add more members as desired by incrementing the index
+```
+
+</TabItem>
+
+<TabItem value="helm-values">
+
+```yaml
+orchestration:
+  security:
+    initialization:
+      defaultRoles:
+        <role>:
+          users:
+            - <username>
+          clients:
+            - <client id>
+          groups:
+            - <group id>
+          mappings:
```

Comment:
```suggestion
          mappingrules:
```

Could also be `mappingRules`. But no kebab case `mapping-rules` as long as github.com/camunda/camunda/issues/36382 isn't fixed.